tests/pkg_etl/Makefile.ci

```diff
@@ -0,0 +1,2 @@
+BOARD_INSUFFICIENT_MEMORY := \
+    nucleo-l011k4
```

Comment:
```suggestion
    nucleo-l011k4 \
    #
```

Reply:
Changed it. Could you explain what this is for?

Reply:
Adding a new board after the nucleo-l011k4 will only change one line. That's minor but cleaner.